src/lib.rs

```diff
@@ -99,6 +100,112 @@ where
     namespace: BTreeMap<String, AmlValue>,
 }
 
+/// The pointer to the EBDA (Extended Bios Data Area) start segment pointer
```

Comment:
Could we also move these constants and the two functions into a `rsdp_search.rs` module and then re-export `search_for_rsdp_bios` from here, just to keep things a bit cleaner

Reply:
Yeah sure